en/starter/basic-routing.md

```diff
@@ -1,68 +1,73 @@
 ---
 layout: page
-title: Express basic routing
+title: Building a Node.js Express Server: Basic Routing
 menu: starter
 lang: en
 redirect_from: "/starter/basic-routing.html"
 ---
 
-# Basic routing
 
-_Routing_ refers to determining how an application responds to a client request to a particular endpoint, which is a URI (or path) and a specific HTTP request method (GET, POST, and so on).
+# Basic Routing with Express: A Step-by-Step Guide
 
-Each route can have one or more handler functions, which are executed when the route is matched.
+Welcome to the world of building web servers with Node.js and Express! In this guide, we'll explore the fundamentals of routing in Express, which is crucial for handling client requests effectively.
 
-Route definition takes the following structure:
+Understanding Routing
```

Comment:
This should be a heading

Reply:
Thankyou, I noted all the points 
Yes I am used to writing blogs and storytelling.